src/bin.ts

```diff
@@ -257,6 +266,10 @@ const makePluggableBin = async (): Promise<Bin> => {
   bin = bin.action(async (options, files) => {
     const { run } = await import("./index.js");
     const baseOptions = await normalizeOptions(options, files);
+    const stderr = new Logger(baseOptions.logLevel, "stderr");
+    for (const pluginName of failedPlugins) {
+      stderr.warn(`Cannot find package '${pluginName}'`);
```

Comment:
not sure if this is the best place to do these warnings

if you have a better idea, im all ears

Reply:
Is that matching prettier v3's behavior? I feel like if a problem that is needed is not found we should exit, not log an error and do something that the user probably didn't intend us to do, presumably assuming that their plugins would be loaded properly.

Reply:
i think you're right. iirc prettier continues execution and logs a warning/error about the plugin not existing

but i think we should just exit like you say